src/secp256k1.c

```diff
@@ -633,17 +671,6 @@ int secp256k1_ec_pubkey_negate(const secp256k1_context* ctx, secp256k1_pubkey *p
 }
 
 
-static int secp256k1_ec_seckey_tweak_add_helper(secp256k1_scalar *sec, const unsigned char *tweak) {
```

Comment:
Ah good point. The reason I moved this was just because it wasn't declared yet when I #included eccommit_impl.h. I'll change it to just add a forward declaration to ecccommit_impl.h.